src/app/features/courses/courses-list/courses-list.component.html

```diff
@@ -13,20 +13,28 @@
       <button class="add-btn">Add Course</button>
     </div>
   </div>
-  <div class="matches">{{courses.length}} COURSES FOUND</div>
-  <div class="courses-list">
-    <app-courses-list-item 
-      class="list-item"
-      *ngFor="let course of courses | slice: 0: limit"
-      [course]="course"
-      (removeCourse)="deleteCourse($event)">
-    </app-courses-list-item>
-  </div>
-  <div class="load-more"
-    *ngIf="limit <= courses.length">
-    <button class="load-btn"
-      (click)="loadMore()">
-      LOAD MORE
-    </button>
+  <div class="matches">{{searchedCourses.length}} COURSES FOUND</div>
+  <div *ngIf="searchedCourses.length; else noCourses">
```

Comment:
we could use ng-container instead of <div>. It could help avoid unnecessary tag if we have > 0 courses.
<ng-container *ngIf="searchedCourses.length; else noCourses">